src/pages/api/meeting.ts

```diff
@@ -57,6 +124,12 @@ const handler = generateMethodRoute(
       }
       return deletedDoc;
     },
+  },
+  {
+    cors: {
+      methods: ["POST"],
+      origin: [/http:\/\/localhost:\d*/, "https://bog-video.netlify.app/"],
```

Comment:
Would it be better to have these urls in `utils/urls` and import them in case we have to update these later?

Reply:
I think that file is more for in-app urls. Maybe as environment variables? Since they're only in in one place I wasn't sure if I should make them as environment variables or not